src/constant/index.js

```diff
@@ -358,28 +358,28 @@ const SCOPES = [
     product: "LiveChat",
   },
   {
-    id: "properties.license.value--my.rw",
```

Comment:
Changed the mistake - a dot was used instead of a colon. Changed in all scopes below as they had the same mistake.